src/constants/consentTypes.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Consent types
+ *
+ * @constant {Object} CONSENT_TYPES – The types of consent.
+ */
+export const CONSENT_TYPES = Object.freeze({
+  TOS: 'tos',
+  ASSENT: 'assent',
+  ASSENT_ES: 'assent-es',
```

Comment:
probably needed for later:
```suggestion
  ASSENT_ES: 'assent-es',
  CONSENT: 'consent',
  CONSENT_ES: 'consent-es'
```

We have a bunch more (see legal collection on the Roar Admin database), but those are the ones more typically used, others are : 

- assent-behaivoral-eye-tracking-es
- assent-behavioral-eye-tracking
- assent-clinic-adobe
- assent-video-audio-eye-tracking
- assent-video-audio-eye-tracking-es
- assent-video-audio-recording
- assent-video-audio-recording-es
- consent-behaivoral-eye-tracking-es
- consent-behavioral-eye-tracking
- consent-clinic-adobe
- consent-video-audio-eye-tracking
- consent-video-audio-eye-tracking-es
- consent-video-audio-recording
- consent-video-audio-recording-es

Reply:
I guess we do not this for now, just the Spanish ones and TOS

Reply:
Thanks for providing the list! I noticed there were more consent types but since I didn't touch / see another part of the code that explicitly checked for those, I omitted them (for now) from the constant. However, it definitely does not hurt to have them there, so added all of those as requested by you and @richford.

Only question are the names of the following actually correct or just accidental typos? 
* `assent-behaivoral-eye-tracking-es`
* `consent-behaivoral-eye-tracking-es`

Reply:
Can we correct it in the database(s)?

Reply:
@Emily-ejag @richford just following up – do we want to correct in the db or should I introduce them with a typo in the constants?

Reply:
Yes, let's correct in the DB. @Emily-ejag can you please do that?

Reply:
Hmm. That would be effort than I originally thought. Because this issue fixes a bug and is more urgent, @maximilianoertel , can you use the incorrect names for now? That is, can you incorporate the typos in the source code so that the dashboard code is consistent with the database?

Then, we will add another task to fix the typos. But this will involve updating all of the collection names and also all of the user docs for users who have assented/consented to those documents.

Reply:
Done @richford